### DISCUSSION TITLE
Building a Rust parser using Pest and PEG

### DISCUSSION THREAD

**Discussion by tomtau** [pest-parser/pest#791] [Show and tell]
@YJDoc2 wrote this nice walkthrough https://blog.logrocket.com/building-rust-parser-pest-peg/ 👏

**Comment by YJDoc2** [comment]
Hey, thanks for mentioning it here! I had a lot of fun when working with this amazing project as I was writing the post. I hope the post will be useful for people  to get started with pest :smile:
